pkgs/by-name/ku/kubectl-rook-ceph/package.nix

```
@@ -0,0 +1,52 @@
+{
+  buildGoModule,
+  fetchFromGitHub,
+  lib,
+  nix-update-script,
+}:
+
+buildGoModule (finalAttrs: {
+  pname = "kubectl-rook-ceph";
+  version = "0.9.3";
+
+  src = fetchFromGitHub {
+    owner = "rook";
+    repo = "kubectl-rook-ceph";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-stWuRej3ogGETLzVabMRfakoK358lJbK56/hjBh2k2M=";
+  };
+
+  vendorHash = "sha256-fB3S946nv1uH9blek6w2EmmYYcdnBcEbmYELfPH9A04=";
+
+  postInstall = ''
+    ln -s $out/bin/cmd $out/bin/kubectl-rook-ceph
```

```suggestion
    mv $out/bin/cmd $out/bin/kubectl-rook-ceph
```

If `cmd` is not needed.

---

Thanks, fixed!